test/lyapunov.jl

```diff
@@ -175,51 +173,55 @@ end
     lorenztr = tr(jacobian(dq0TN)) #trace of Lorenz system Jacobian matrix
     @test lorenztr == -(1+σ+β)
 
-    xw, λw = lyap_taylorinteg(lorenz!, q0, t0:1.0:tmax, _order, _abstol; maxsteps=2)
+    trange = t0:1.0:tmax
+    xw, λw = lyap_taylorinteg(lorenz!, q0, trange, _order, _abstol; maxsteps=5)
     @test xw[1,:] == q0
-    @test size(xw) == (length(t0:1.0:tmax), 3)
-    @test size(λw) == (length(t0:1.0:tmax), 3)
+    @test size(xw) == (length(trange), 3)
+    @test size(λw) == (5+1, 3)
```

Comment:
Why not writing this test as `@test size(λw) == (6, 3)`, and add a comment (at the end of the line?) that 6 = maxsteps+1.

Reply:
Thanks, I just addressed both lines